config/routes.rb

```
@@ -154,21 +154,23 @@
           patch '/:id/:authorization', to: 'participants#update_authorization'
-          delete '/:id', to: 'participants#destroy'
-        end
-      end
-
-      resources :student_teams, only: %i[create update] do
-        collection do
-          get :view          
-          get :mentor
+          delete '/:id', to: 'participants#destroy'
+        end
+      end
```

RuboCop (1.86.1)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)